doc/distributions/nixos.md

```diff
@@ -6,7 +6,8 @@ followed.
 
  - https://nixos.wiki/wiki/NixOS_on_ARM/UEFI
 
-They will refer back to Tow-Boot as a source of *Platform Firmware*.
+They will refer back to Tow-Boot as a source of *Platform Firmware*. You will
+need to know that Tow-Boot cannot write to EFI variables.
```

Comment:
(Note to self) Once the operating system is booted, writing to EFI variables will not be possible.